scss/mixins/_spinners.scss

```diff
@@ -0,0 +1,121 @@
+// Spinner variants
+//
+// Easily pump out default styles, sizes, styling variants
+
+@keyframes spinner {
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@mixin spinner-size($size, $width: 0) {
+  $inside-size: ($size - 2 * $width);
+
+  min-width: $size;
+  min-height: $size;
+
+  &:before,
+  &.spinner-fallback .spinner-fallback-spin {
+    width: $inside-size;
+    height: $inside-size;
+    margin-top: (-$inside-size / 2);
+    margin-left: (-$inside-size / 2);
+    border-width: $width;
+  }
+}
+
+@mixin spinner-circle-variant($color-first, $color-second) {
+  &:not(.spinner-fallback):before,
+  &.spinner-fallback .spinner-fallback-spin {
+    border-style: solid;
```

Comment:
Properties should be ordered border-color, border-style, border-top-color, border-radius